src/dcap_provider.cpp

```diff
@@ -104,16 +104,16 @@ static std::string get_collateral_version()
 
     if (collateral_version.empty())
     {
-        log(SGX_QL_LOG_WARNING,
+        log(SGX_QL_LOG_INFO,
             "Using default collateral version '%s'.",
             default_collateral_version.c_str());
         return default_collateral_version;
     }
     else
     {
-        if (!collateral_version.compare("v1") &&
-            !collateral_version.compare("v2") &&
-            !collateral_version.compare("v3"))
+        if (collateral_version.compare("v1") &&

```

Comment:
Nit: Wondering if this would make more sense as an else if block and the final else block being the safe return rather than an if within the else block.